receipt_parser_core/receipt.py

```diff
@@ -108,11 +108,14 @@ def parse_items(self):
         items = []
         item = namedtuple("item", ("article", "sum"))
 
-        ignored_words = self.config.sum_keys
-        for word in self.config.ignore_keys:
-            ignored_words.append(word)
+        ignored_words = self.config.ignore_keys
+        stop_words = self.config.sum_keys
 
         for line in self.lines:
+            for stopWord in stop_words:
```

Comment:
`stopWord` -> `stop_word` as per [PEP8](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names) 😉

Reply:
Yes! I will fix this.